tests/dummy/app/controllers/application.js

```diff
@@ -93,8 +96,14 @@ class ApplicationController extends Controller {
 
     @action
     alertData(row) {
-        const data = row.rowValue;
-        alert(`${data.name} is ${data.age} years old.`);
+        let expandedRows = this.expandedRows.concat([]);
+        if (expandedRows.includes(row.id)) {
+            const indAt = expandedRows.indexOf(row.id);
+            expandedRows.splice(indAt, 1);
+        } else {
+            expandedRows = this.expandedRows.concat([row.id]);
+        }
+        this.expandedRows = expandedRows;
     }
```

Comment:
I would add an example of how to trigger the expanded row toggling from a custom cell component, as I think thats probably the more common pattern we'll use that for (and is how gabriella will be using this in J&R)